components/kyma-operator/cmd/operator/main.go

```diff
@@ -104,6 +116,10 @@ func main() {
 	internalInformerFactory.Start(stop)
 
 	installationController.Run(stop)
+
+	if closeFn != nil {
```

Comment:
I recommend to **always** return a function, instead of having `nil` check. In the case when the function shouldn't do anything, then - well, just provide the function that doesn't do anything.NOOP.

Reply:
done